### DISCUSSION TITLE
Add a "shortDescription" field to the Product entity

### DISCUSSION THREAD

**Discussion by BenVuti** [vendurehq/vendure#1027] [General]
I've added a short description customField to my Produt entity.
This is in order to display a short text in my product cards.
I think many could make use of such a field if it was natively included in the product entity.

**Comment by michaelbromley** [comment]
Hi,

In general my approach is to keep the default fields to a minimum, because each implementation tends to have very different needs. For this reason I tried to make it as easy as possible to add the needed fields via customFields. So in the case you described, it is working as intended.

If there is a lot of demand for certain fields to be added by default, I would consider adding them but that would only happen in the next major version (v2.0) since it would require a breaking schema change.
